### DISCUSSION TITLE
Welcome to MiniUtilities Discussions!

### DISCUSSION THREAD

**Discussion by OneLemonyBoi** [OneLemonyBoi/MiniUtilities#80] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by holyaurora21** [comment]
add golden lasso? https://ftb.fandom.com/wiki/Golden_Lasso_(Extra_Utilities_2)

**Comment by DeadSlayer54** [comment]
Is there a discussion for ideas besides the generators?

**Comment by xing2048** [comment]
"Angel Ring" there is an bug ?
Inability to fly normally.
